jcl/src/java.base/share/classes/java/lang/ref/Reference.java

```diff
@@ -196,24 +192,19 @@ public boolean isEnqueued () {
 	@NotCheckpointSafe
 	/*[ENDIF] CRIU_SUPPORT */
 	boolean enqueueImpl() {
-		final ReferenceQueue tempQueue;
-		boolean result;
 		T tempReferent = referent;
 		synchronized(this) {
 			/* Static order for the following code (DO NOT CHANGE) */
-			tempQueue = queue;
+			final ReferenceQueue tempQueue = queue;
 			queue = null;
-			if (state == STATE_ENQUEUED || tempQueue == null) {
+			if ((tempQueue == null) || (state == STATE_ENQUEUED)) {
 				return false;
 			}
-			result = tempQueue.enqueue(this);
-			if (result) {
-				state = STATE_ENQUEUED;
-				if (null != tempReferent) {
-					reprocess();
-				}
+			tempQueue.enqueue(this);
+			if (null != tempReferent) {
```

Comment:
Lines 200 and 204 should be consistent: the `null` (or the constant) go on the left or the right, but not a mix of both, please.

Reply:
Updated.

Reply:
`STATE_ENQUEUED` is also a constant: it should go on the left just like `null`.